apps/docs/docs/tutorials/api-sdk/credential-group.md

```diff
@@ -0,0 +1,312 @@
+---
```

Comment:
I think this file can be called `groups` since it covers group operations with the API SDK.